### PR TITLE
change prometheus-operator repo

### PR DESCRIPTION
**Pull request by vsinghal13**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -786,9 +786,9 @@ fluent-bit:
 
     @INCLUDE fluent-bit-output.conf
 
-## Configure prometheus-operator
-## ref: https://github.com/helm/charts/blob/master/stable/prometheus-operator/values.yaml
-prometheus-operator:
+## Configure kube-prometheus-stack
+## ref: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml
+kube-prometheus-stack:
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:791]: Do we want to use `kube-prometheus-stack`? We could use `alias` in the requirements to keep backward compability in the configuration

**Review by frankreno — APPROVED** [review]
+1 to aliasing, otherwise LGTM

File: deploy/helm/sumologic/requirements.yaml [hunk 2]
```diff
@@ -3,9 +3,9 @@ dependencies:
     version: 2.8.14
     repository: https://kubernetes-charts.storage.googleapis.com/
     condition: fluent-bit.enabled,sumologic.logs.enabled
-  - name: prometheus-operator
-    version: 8.13.8
-    repository: https://kubernetes-charts.storage.googleapis.com/
+  - name: kube-prometheus-stack
+    version: 9.3.4
+    repository: https://prometheus-community.github.io/helm-charts
     condition: prometheus-operator.enabled,sumologic.metrics.enabled
```
Comment (perk-sumo) [inline deploy/helm/sumologic/requirements.yaml:9]: Shouldn't we add an alias as well?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/requirements.yaml:9]: you mean this?
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/requirements.yaml#L9

Comment (vsinghal13) [reply deploy/helm/sumologic/requirements.yaml:9]: Alias was added in separate PR: #910

**Pull request merged**